<launch>

<node pkg="depth_flight_controller" type="image_clipper" name="image_clipper" output="screen"/>

<node pkg="depth_flight_controller" type="c_space_expander" name="c_space_expander" output="screen"/>

<node pkg="depth_flight_controller" type="horizon_plotter" name="horizon_plotter" output="screen"/>

<node pkg="depth_flight_controller" type="image_prep" name="image_prep" output="screen"/>

<node pkg="rqt_image_view" type="rqt_image_view" name="rqt_image_view" output="screen"/>

</launch>